README.md

```diff
@@ -1,2 +1,32 @@
-# terraform-is
-Terraform scripts for provision IS deployments on Azure cloud
+# Installation
```

Comment:
@rasanjaya85 since this is the main `README`, shall we provide the name of the repository here and add `Installation` as part of the contents of this `README`? For example, something like `Terraform Resources for WSO2 Identity Server`.